### ISSUE TITLE
ZK Users path missing in Constants

### ISSUE THREAD

**Issue by dlmarion** [apache/accumulo#3120]
Create a ZUSERS path variable in Constants, use in ZKAuthenticator at https://github.com/apache/accumulo/blob/main/server/base/src/main/java/org/apache/accumulo/server/security/handler/ZKAuthenticator.java#L56 and maybe other locations.

**Comment by ddanielr** [comment]
@dlmarion :+1:

**Comment by ddanielr** [comment]
So is my understanding correct? 
`ZKUserPath` is scoped correctly and does not need to be modified.

1.  Define a `/users` var in Constants called `ZKUSERS` for use in all ZKuser paths: 
https://github.com/apache/accumulo/blob/main/core/src/main/java/org/apache/accumulo/core/Constants.java 

2. Find all instances of hardcoded `/users` path in use by any `ZKUser` operations and replace with new `ZKUSERS` Constant value.

Intended fix should look like `ZKUserPath = Constants.ZROOT + "/" + context.getInstanceID()  + Constants.ZUSERS;`

**Comment by dlmarion** [comment]
Yes. Might be useful to look for any "/users" string in the code and evaluate whether it should use the new constant. There may be some tests.

> replace with new ZKUSERS Constant value

I think should be `ZUSERS` based on convention.

**Comment by ctubbsii** [comment]
The `/users` is specific to the ZKAuthenticator, and not, as a general rule, part of Accumulo at large, with other authenticators. That said, the authentication stuff probably needs work, and any work I can imagine would have a `/users` for at least the built-in local superuser ("root"), so it's probably okay to have this exist for all authenticators. Right now, though, only ZKAuthenticator and those that might be derived from it make use of this path, so it's still very specific to those.
